### PR TITLE
Allow to add links with label in Crowdin

### PR DESCRIPTION
**Pull request by fertandil87**
images in the ticket https://globalfishingwatch.atlassian.net/browse/VV-337

### CODE REVIEW

File: apps/vessel-history/features/profile/components/GuideAndVideo.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { event as uaEvent } from 'react-ga'
+import { useTranslation } from 'react-i18next'
+import styles from './Info.module.css'
+
+
+interface GuideAndVideoProps {
+    source?: string
```
Comment (rdgfuentes) [inline apps/vessel-history/features/profile/components/GuideAndVideo.tsx:8]: Why not using `label` for this prop, it seems more straight forward? What do you think?

File: apps/vessel-history/features/profile/components/Faq.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { event as uaEvent } from 'react-ga'
+import { useTranslation } from 'react-i18next'
+import styles from './Info.module.css'
+
+
+interface FaqProps {
+    source?: string
```
Comment (rdgfuentes) [inline apps/vessel-history/features/profile/components/Faq.tsx:8]: I'd use `label` for this prop. WDYT?

File: apps/vessel-history/features/profile/components/GuideAndVideo.tsx (hunk 1 above)
Comment (fertandil87) [reply apps/vessel-history/features/profile/components/GuideAndVideo.tsx:8]: because I was thinking the source of where is clicked

**Review by rdgfuentes — APPROVED** [review]
👍

**Pull request merged**